src/main/groovy/io/gatling/gradle/GatlingPlugin.groovy

```diff
@@ -61,6 +64,13 @@ final class GatlingPlugin implements Plugin<Project> {
         }
     }
 
+    private void registerEnterpriseDeployTask(Project project, TaskProvider<GatlingEnterprisePackageTask> gatlingEnterprisePackageTask) {
+        project.tasks.register(ENTERPRISE_DEPLOY_TASK_NAME, GatlingEnterpriseDeployTask.class) {
+            inputs.files gatlingEnterprisePackageTask
+            dependsOn(gatlingEnterprisePackageTask)
```

Comment:
1. not the only dependency I guess
2. does that answer my previous question? (I'm astonished to see this 2 informations so far from one another)

Reply:
That's my only dependency for this task : I need gatlingEnterprisePackage executed, and the file produced (the simulation jar) is injected as input of my task.

Actually it's my understanding (and other task, like upload or enterpriseStart are implemented that way)

Reply:
After discussion : Cache disabled for this task